src/GroupManager.php

```diff
@@ -123,10 +229,39 @@ public function removeGroup($entity_type_id, $bundle_id) {
   }
 
   /**
+   * Refreshes the locally stored data.
+   *
+   * Call this after making a change to a relationship between a group type and
+   * a group content type.
+   */
+  public function refresh() {
```

Comment:
maybe this should be a simple reset() - and the next call will just get new values?